### PR TITLE
[Travis] Run tests on PHP 7.1

### PR DESCRIPTION
**Pull request by Naktibalda**

### CODE REVIEW

**Comment by Naktibalda** [comment]
Phalcon is not compatible with PHP 7.1 yet
https://github.com/phalcon/cphalcon/issues/12055

File: .travis.yml [hunk 1]
```diff
@@ -8,15 +8,15 @@ php:
 
 matrix:
   include:
+    - php: hhvm
```
Comment (DavertMik) [inline .travis.yml]: +1 more reason to move HHVM to separate CI. 
I will try to work on Wercker to support all current test cases and then we could try to move it out. 
I think moving that away would improve the overall stability of Travis builds

**Comment by DavertMik** [comment]
Please look into my PR: https://github.com/Codeception/Codeception/pull/3809
I didn't reinvent the wheel, I moved most of your commands for HHVM to wercker.yml and made them running there. Also I granted you admin rights on wercker ci.

File: .travis.yml (hunk 1 above)
Comment (DavertMik) [inline .travis.yml]: HHVM took 35 minutes to run the tests 0.o
I assume it's not a travis issue, they are slow on wercker too. Probably the slowdown happens for filesystem operations, cli tests run painfully slow (tried them locally and on wercker too)

**Pull request merged**